ExCSystem/settings/development.py

```diff
@@ -16,3 +16,5 @@
 EMAIL_HOST = 'localhost'
 EMAIL_PORT = 1024
 
+# Base address of where the page is available
+WEB_BASE = "http://127.0.0.1:8000"
```

Comment:
@TomekFraczek shouldn't this be localhost?

Reply:
Not sure. I know on my machine it really likes being on 127.0.0.1. That being said, this can/should be adjusted in the dev environment by each person.

Ill and a more saying that's the case.